app/javascript/components/App.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { BrowserRouter as Router, Route } from 'react-router-dom';
+import Greeting from './Greeting';
+
+function App() {
+  return (
+    <Router>
+      <div>
+        <Route exact path="/" component={Greeting} />
+      </div>
+    </Router>
+  );
+}
+
+export default App;
```

Comment:
- [ ] Your app component is not called anywhere in your application, please make sure to import it so it will be used.